apps/console/src/features/identity-providers/data/identity-provider-templates/templates/saml-identity-provider/saml-idp-wizard-help.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { Heading } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement } from "react";
+import { Divider } from "semantic-ui-react";
+
+type Props = TestableComponentInterface;
```

Comment:
Let's rename to `SAMLIDPWizardHelpPropsInterface`.We can do this in a subsequent PR.
```suggestion
type SAMLIDPWizardHelpPropsInterface = TestableComponentInterface;
```